ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -62,6 +62,29 @@ FCU_DFSS_DFVSC_DXRC_DFRMM:
   - DXRC
   - DFRMM
 
+FCU_DFSS_DFVSC_DSP_CHWDC_HWDC_CO2M:
+  description: "Hydronic variable speed fan coil with heating and cooling (zone temp control & CO2 monitoring)."
```

Comment:
The description identifies zone temp control but the type is implementing `CHWDC` and `HWDC` - please clarify the description

Reply:
thanks, updated the descriptions, PTAL.